control/cli.py

```diff
@@ -26,8 +26,6 @@
 from .utils import GatewayUtils
 from .utils import GatewayEnumUtils
 
-BASE_GATEWAY_VERSION="1.0.0"
```

Comment:
This has nothing to do with protobuf cleanup and it removes an existing behavior. As we broke the protocol between the CLI and the gateway in version 1.0.0 we added this test to give an error in case the user try to mix pre-1.0.0 version with a 1.0.0 one. This combination wouldn't work so we wanted to give a clear message about it. If you think this should be removed you need to discuss it with @caroav . It can't be done as part of a cleanup.

Reply:
@gbregman thank you for review.

I was not sure about the functionality of `BASE_GATEWAY_VERSION` or about its value. 

I could imagine a logic of cli which refuses to work with gw which reports incompatible version, for instance verifying compatibility behind the scene  before issuing the requested cli command like "add namespace", however in current implementation where is is used as a warning in `def gw_get_info` looks strange. 

Regarding the value, why don't we just compare the gateway version with cli one, why this constant is required?

Please share you thoughts.

Reply:
@baum  I suggest to detach this from the proto-buf cleanup PR. Then we can continue discussing this in another issue. Otherwise this discussion will hold this PR in the air for too long.

Reply:
@caroav would it be acceptable 
1. in  gw_get_info function, restore the compare logic, however compare gateway version to CLI one
2. remove the constant

Reply:
@baum we do compare the CLI and gw versions and issue a warning if they don't match. Anyway we needed something which will work even in cases there CLI-gw protocol is broken. So, I made sure the fewtching of the version still works after the protocol changes so we could issue the warning about the broken protocol.